plugins/com.google.cloud.tools.eclipse.appengine.newproject/plugin.properties

```diff
@@ -7,5 +7,10 @@ gcp.category = Google Cloud Platform
 standard.wizard.name = Google App Engine Standard Java Project
 standard.wizard.description = Creates an App Engine Standard Java project.
 
-flex.wizard.name=Google App Engine Flexible Java Project
-flex.wizard.description=Creates an App Engine Flexible Java project.
+flex.war.wizard.name=Google App Engine Flexible Java WAR Project
+flex.war.wizard.description=Creates an App Engine Flexible Java project whose deploy artifact is a \
+ web application archive (WAR).
+
+flex.jar.wizard.name=Google App Engine Flexible Java JAR Project
+flex.jar.wizard.description=Creates an App Engine Flexible Java project whose deploy artifact is a \
+ runnable "fat" Java archive (JAR).
```

Comment:
I think just "fat" JAR is fine; i.e. don't need to spell out Java archive. Better yet

"Creates an App Engine Flexible Java project for the Java runtime" though I do wish the App Engine team would give these runtimes better names.